src/Classes/FinalRule.php

```diff
@@ -120,7 +120,12 @@ private function isWhitelisted(Node $node): bool
             return false;
         }
 
-        if (\is_int(\preg_match_all('/@(\S+)(?=\s|$)/', $docComment->getReformattedText(), $matches))) {
+        $reformattedComment = $docComment->getReformattedText();
+        if (!\is_string($reformattedComment)) {
```

Comment:
`getReformattedText` now returns `string|mixed`, so we need to make sure it's a string before we pass it to `preg_match_all` which strictly expects a `string`.